internal/native/cgo_linux.go

```
@@ -268,8 +269,12 @@ func uiObjAddFlag(objName string, flag string) (bool, error) {
 	defer C.free(unsafe.Pointer(objNameCStr))
 	flagCStr := C.CString(flag)
 	defer C.free(unsafe.Pointer(flagCStr))
-	C.jetkvm_ui_add_flag(objNameCStr, flagCStr)
-	return true, nil
+
+	ret := C.jetkvm_ui_add_flag(objNameCStr, flagCStr)
```

Handle the error returns correctly (was silently ignored)